plugin/manager_linux.go

```diff
@@ -27,10 +28,27 @@ func (pm *Manager) enable(p *v2.Plugin, force bool) error {
 	p.Lock()
 	p.Restart = true
 	p.Unlock()
+
+	mnt := filepath.Join(p.Rootfs, "mnt")
```

Comment:
If we're going to do this, then yes. My plugin just failed due to this as well. But I'm not sure we should really do this. Plugins can add the mount if they want to, no? Or is this just so they plugin author doesn't have to ask permission from the user since it's a guarantee?